client/app/src/accounts/account.controller.js

```diff
@@ -613,6 +619,15 @@
       return balance * price
     }
 
+    self.toggleBitcoinCurrency = function (force) {
+      self.bitcoinToggle = force !== undefined ? force : !self.bitcoinToggle
```

Comment:
I think it'd be a good idea to rename bitcoinToggle to something more indicative of it being an active or inactive bool value (ex: btcValueActive). Also, you may as well use a ternary instead of the if/else.